README

```diff
@@ -1 +1,6 @@
-Hello World!
\ No newline at end of file
+Hello World!
+$ mkdir ~/Hello-WorldCreates a directory for your project called "Hello-World" in your user directory
+$ cd ~/Hello-WorldChanges the current working directory to your newly created directory
+$ git initSets up the necessary Git files
+Initialized empty Git repository in /Users/your_user_directory/Hello-World/.git/
```

Comment:
Lets create another comment and see the position.

Reply:
I dont agree to this comment, so i will reply.